chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,343 @@
+| Voxpopuli      | 543         | European Parliament recordings        | Spontaneous            | ❌                    | ❌                         | Non-native English speakers                       |
+| GigaSpeech     | 10000       | Audiobook, podcast, youtube           | Narrated & spontaneous | ❌                    | ✅                         | Robustness over multiple domains                  |
+| SPGISpeech     | 5000        | Financial meetings                    | Narrated & spontaneous | ✅                    | ✅                         | Fully formatted transcriptions                    |
+
+
+Let’s take an example where we want to train a speech recognition system to transcribe speech on the topic of machine learning (highly relevant!). Our speech is going to be narrated (scripted), and we’re not worried about punctuation or casing on our text output. From our reference table, it looks like TED-LIUM is a good choice of dataset!
+
+[//]: # On review, this sub-section might be too involved. Happy to omit it and dive straight into how to load a dataset.
+
+### How to load an audio dataset
+Before we can handle any audio datasets, we need to make sure we have the right dependencies installed. Specifically, we'll need the "Audio" feature from datasets: https://huggingface.co/docs/datasets/installation#audio. This should take care of the main Python packages we need to read audio files from byte form and convert them to arrays.
+
+Right! Now we're ready to go ahead and download our data. For this tutorial, we'll use the [_LibriSpeech ASR_](https://huggingface.co/datasets/librispeech_asr) corpus. LibriSpeech is one of the most popular datasets for benchmarking speech recognition systems in both academia and industry. It consists of approximately 1000 hours of narrated audiobooks collected from the (LibriVox)[https://librivox.org/] project.
+
+
+Let’s take a look at the all-time leaderboard on Papers with Code to get a feel for numbers: https://paperswithcode.com/sota/speech-recognition-on-librispeech-test-clean
+We can see that there’s been tremendous progress on the LibriSpeech benchmark in the last three years, with many of the top spots being occupied by Wav2Vec2 or Wav2Vec2-like systems! As mentioned in Section 4, Wav2Vec2 revolutionised the field of speech recognition by introducing an effective pre-training regime, much the same way as BERT revolutionised NLP. We can see that the word error rates (WERs) for this dataset are extremely low! We’ll cover WER in more detail in this section, but for now we can see that the best performing system achieves a WER of just 1.4%, or a word accuracy rate of 100 - 1.4 = 98.6%! This is very impressive! We’ll see how close we can get with our system 🤗
+
+Now that we've got better idea about the dataset that we're working with, let's download it using the `load_dataset()` function:
+
+```python
+from datasets import load_dataset, DatasetDict
+
+raw_datasets = DatasetDict()
```

Comment:
Since `datasets` downloads all splits, do we lost a lot of time by simply running?

```
raw_datasets = load_dataset("librispeech_asr", "clean")
```

Reply:
Switched to streaming mode: https://github.com/huggingface/course/pull/294/commits/629f03bb30d32b93172fe3f138fea25ef9a7d757

The dataset is now downloaded on-the-fly, chunk by chunk